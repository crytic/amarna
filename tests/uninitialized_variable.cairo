func bar() -> (res):
    alloc_locals
    local x
    local y = 2
    local z
    let z = 1
    local b
    assert b = 2
    local a
    a = 3
    assert x * x = 25
    return (res=x)
end