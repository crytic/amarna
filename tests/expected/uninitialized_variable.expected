[arithmetic-mul] in uninitialized_variable.cairo:11:12
[dead-store] in uninitialized_variable.cairo:10:5
[dead-store] in uninitialized_variable.cairo:4:11
[dead-store] in uninitialized_variable.cairo:5:11
[uninitialized-variable] in uninitialized_variable.cairo:3:11
[unused-function] in uninitialized_variable.cairo:1:1